### DISCUSSION TITLE
Are there any plans to implement a tui to the installer?

### DISCUSSION THREAD

**Discussion by Relms12345** [archlinux/archinstall#191] [Q&A] [answered]
I personally would love to see smth like a ncurses-backed TUI to the installer so that it can be easier for ppl to choose options imo. Also if someone messes up an option they could go back and redo that one question instead of doing ctrl+c and restarting the entire installer.

**Comment by Torxed** [comment, marked as answer]
The ability to go back and fourth between options are being considered and designed, as it has been an issue (#129).
I'll begin my next statement by saying yes - the installer isn't really looking like a factory fresh lambo, more like a 1980's LADA.. But I personally don't like the look and feel of ncurses, it would solve the issue but I'm looking in to other options that may feel a bit more inuitive to code and use.

But the fact remains the same, being able to change options will be fixed! Just need some time :)

**Reply by Relms12345** [reply]
Ok sounds good to me. Thanks!
